Kanban/QSFU_kanban.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+#==============================================================================
+
+from redminelib import Redmine
+import pdb
+import datetime
+import logging
+import json
+import statistics
+import configparser
+
+config = configparser.ConfigParser()
+config.read('config_kanban.ini')
+
+URL = config['API']['URL']
+KEY = config['API']['ApiKey']
+
+redmine = Redmine(URL, key=KEY)
+
+prjlist = ['suseqa', 'openqav3', 'openqatests']
+taglist = ['y', 'epic', 'saga', 'fate']
+cycleList = []
+leadList = []
+
+id_codes = {
+        'New' : '1',
+        'In Progress' : '2',
+        'Resolved' : '3',
+        'Feedback' : '4',
+        'Workable' : 12
+        }
+
+
+def went_in_progress(jsn):
+    if jsn.get("name", "invalid_default") == "status_id":
+        if jsn["new_value"] == id_codes.get("In Progress"):
+            return jsn["old_value"] in [id_codes["Workable"], id_codes["New"]]
```

Comment:
I remember that this section was previously wrapped in a try-except-block to catch missing keys inside the dict.
While you changed the value-access in L#35, the script could now easily break if the key "name" is available (`if` entered) but any of the subsequently ones is missing ("new_value" or "old_value"). Better use the getter here too.

Reply:
In my understanding it *should* be impossible that we have a status_id change without "new_value" and "old_value"
So as soon as we have a "status_id" there must be the other fields